Analytics/resources/sendgrid_management.py

```diff
@@ -69,27 +70,23 @@ def post(self) -> (dict, int):
                 SendgridHelper.handle_sendgrid_response(sendgrid_response)
 
             if result["api_key"]:
-                is_replaced, reason = self.replace_sendgrid_key(new_api_key,
-                                                                "~",
-                                                                ".bash_profile")
+                is_replaced, reason = self.replace_sendgrid_key(new_api_key)
                 if is_replaced:
-                    os.environ["SENDGRID_API_KEY"] = new_api_key
                     logger.info("Sendgrid API key has been replaced "
                                 "with new key = {}".format(new_api_key))
                     return {"message": "success"}, 200
                 else:
                     return {
-                        "message": "failure",
-                        "reason": reason
-                    }, 422
+                               "message": "failure",
+                               "reason": reason
+                           }, 422
             else:
                 return {"message": "failure", "reason": result["reason"]}, 422
         else:
             return {"message": "Not Authorized"}, 403
 
     @staticmethod
-    def replace_sendgrid_key(new_api_key: str, folder: str,
-                             file: str) -> (bool, str):
+    def replace_sendgrid_key(new_api_key: str) -> (bool, str):
```

Comment:
Remove removed keys from docs

Reply:
docString parameters `folder` and `file` removed